docs/source/installation.rst

```diff
@@ -2,152 +2,125 @@ Installation
 ============
 
 Supported OS
-------------
+++++++++++++
 
-Currently, we only support ``MacOS`` and ``Linux`` operating systems. ``Windows``
-users have the possibility to install and use ``ivadomed`` via
-`Windows Subsystem for Linux (WSL) <https://docs.microsoft.com/en-us/windows/wsl/>`_. The steps below (about updating bashrc) are strongly recommended for MacOS users in the installation process but are optional for Linux users.
+Currently, ``ivadomed`` only supports GPU/CPU on ``Linux`` and ``Windows`` and CPU only on ``macOS`` and `Windows Subsystem for Linux <https://docs.microsoft.com/en-us/windows/wsl/>`_.
 
-Open your bash/zsh script file with editor on your computer.
-
-    If you are using bash shell
-    ::
-
-        vim ~/.bashrc
+Step 1: Setup dedicated python environment
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-    If you are using zsh shell
-    ::
-        
-        vim ~/.zshrc
+You can setup ``ivadomed`` using either Conda or Venv:
 
-Write in your .bashrc/.zshrc file with following line.
+.. tabs::
 
-::
+    .. tab:: Install via ``conda``
 
-    export HDF5_USE_FILE_LOCKING='FALSE'
+        This is the easiest way for personal computers.
```

Comment:
Hmm... I'm wondering, is `conda` really the "easiest way for personal computers"?

Most people won't have conda installed on their machines by default, while a lot of people (e.g. Linux users) will have access to a system Python which they can use to create a virtual environment.

So, for the order of the tabs, I'm wondering if `venv` should be the default tab, while `conda` should be the second tab.

Reply:
Very good point. `conda` was useful for GPU related installation but so far, guide has not really leveraged that much. Switching to `venv`.